src/app/editor/editor.js

```diff
@@ -149,9 +149,9 @@ export default class Editor {
 				preview:
 					root.querySelector(
 						// This one is used on New Issue and Add Comment.
-						'.previewable-comment-form > .js-preview-panel,' +
+						'.previewable-comment-form .js-preview-panel,' +
 						// This one is used on Edit Comment.
-						'.previewable-comment-form > .preview-content' )
+						'.previewable-comment-form > div > .preview-content' )
```

Comment:
Honestly, I don't have the confidence that this div will remain for a long time. It looks super buggy (a div with empty class).

![](https://github.com/ckeditor/github-writer/assets/5353898/426cf6d1-6c69-4855-a728-f1c3bfbd16d7)

Reply:
I'll change it to have two selectors (old, and the one you proposed).